### PR TITLE
Mathcad Wrapper Added

### PR DESCRIPTION
**Pull request by henningjp**
### Description of the Change

Added files to allow:

* Build and install of **PTC Mathcad 15** and **PTC Mathcad Prime** wrappers
* Help documentation through Mathcad 15 Handbook (eBook) and PDF User Guide for Mathcad Prime
* RefProp wrapper function insertion into the Mathcad 15 interface

Built using the REFPROP-headers interface to the installed REFPROP/REFPROP64.DLL.  Mathcad 15 or Prime and NIST RefProp must be installed on the machine.

### Benefits

Allows **PTC Mathcad 15** and **PTC Mathcad Prime** worksheets to make direct calls to the RefProp DLL.

### Possible Drawbacks

Uses the Legacy API (pre-RefProp 10) to access property functions, but can easily be expanded to include the new RefProp 10 API calls, specifically *REFPROPdll()*.

### Verification Process

Code has been extensively tested and verified against the RefProp Excel wrapper and IAPWS-95 documentation, comparing specific state point values for both CO2 and Water, as well as providing full range continuity checks for all fluids.

### CODE REVIEW

**Comment by henningjp** [comment]
@ianhbell I put an externals directory under this Mathcad wrapper directory, but now that I've jumped away to another branch, I see that that may not work.  Should I put the new external (fmtlib) under the main repo?  Can we bump externals\REFPROP-headers to the latest version with out breaking anything else?  If so, I can re-arrange everything to point to the externals in the parent directory.

**Comment by ianhbell** [comment]
I think it is fine to add the the external wherever you want.  Also, I believe there is no downside to bumping REFPROP-headers.  Do you want to do it?  It might be better if I made the update on my side and then you could merge that in your PR? Or?

**Comment by henningjp** [comment]
Let's leave it where it is.  I think I like it "under" the build directory, instead of several levels up.

**Comment by henningjp** [comment]
@ianhbell I'm done tinkering. Tell me what I can do to help you wade through this.

File: README.md [hunk 1]
```diff
@@ -11,12 +11,13 @@
 
 Hint: click on the name to be taken to the appropriate wrapper code
 
-* [Python](wrappers/python)
```
Comment (ianhbell) [inline README.md:14]: Can you please revert this change and add Mathcad somewhere in the list?  They are roughly sorted in terms of importance.

**Review by ianhbell — CHANGES_REQUESTED** [review]
There's quite a lot here to look over....  Can you generate a PDF of all the files in the handbook?

File: .gitmodules [hunk 2]
```diff
@@ -4,3 +4,9 @@
 [submodule "externals/REFPROP-cmake"]
 	path = externals/REFPROP-cmake
 	url = https://github.com/usnistgov/REFPROP-cmake
+[submodule "wrappers/Mathcad/externals/REFPROP-headers"]
```
[the remark this answers is not in the document]
Comment (henningjp) [reply .gitmodules:7]: Probably, but:
1. I'm preparing to add in the REFPROP, REFPROP1, and REFPROP2 calls from version 10 and the one in top level externals is not updated.
2. I rather like having everything "under" the Mathcad directory, in case users want just that directory.

If that's the route you'd rather go, I can certainly reconfigure the build solutions and bump the one in REFPROP-wrappers/externals.  I'll need to add fmt(fmtlib) up there as well.

File: README.md (hunk 1 above)
Comment (henningjp) [reply README.md:14]: No problem.  I just sorted them alphabetically and put Mathcad in the appropriate position.  I'll pull that piece out.  Wasn't aware of an "importance" ranking.  Where should Mathcad go?

**Comment by henningjp** [comment]
The PDF, with only minor differences from Mathcad Prime, is located under wrappers/Mathcad/PrimeDocs/PrimeManual.pdf.

**Comment by ianhbell** [comment]
Wow - I just read over the PDF, that was a crazy amount of work you did.  Should I go ahead and merge and we can handle any issues that come up as they do?

**Comment by henningjp** [comment]
Thanks @ianhbell.  It was spread out over about a decade (started with RefProp 7), so it just crept up.

Let me get those two requested changes fixed up on *Friday* and you can merge it.

File: .gitmodules (hunk 2 above)
[the remark this answers is not in the document]
Comment (henningjp) [reply .gitmodules]: done.

File: README.md (hunk 1 above)
Comment (henningjp) [reply README.md]: Done.

**Comment by henningjp** [comment]
Changes addressed.  Merge away...

**Comment by ianhbell** [comment]
How important is the history here?  Should I squash and merge or just merge?

**Comment by henningjp** [comment]
Squash it.

They were just incremental adjustments to get this initial offering GitHub-ready.  All cosmetic and linkup to actual submodules. History can begin here.

**Comment by ianhbell** [comment]
Squash coming.  Thanks for this very nice contribution.
